### PR TITLE
Clarabranch

### PR DESCRIPTION
**Pull request by clrnb**
svd handles missing bias

### CODE REVIEW

File: peepholelib/tests/sparse_svd_no_bias.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+from numpy.random import randint as ri
+from torch.nn.modules.utils import _reverse_repeat_tuple
+from torch.nn.functional import pad
+importlib.reload(peepholelib.models.svd)
+if __name__ == '__main__':
+    use_cuda = torch.cuda.is_available()
+    cuda_index = torch.cuda.device_count() - 1
+    device = torch.device(f"cuda:{cuda_index}" if use_cuda else "cpu")
+    print(f"Using {device} device")
+
```
Comment (lsrosa) [inline peepholelib/tests/sparse_svd_no_bias.py:15]: given that this test is very similar to the previous one, I would add a boolean variable in line `15`, and use that as the `bias` parameter in line `36`.

So we do not have 2 scripts doing the same thing.

**Review by lsrosa — CHANGES_REQUESTED** [review]
The branch should be rebased onto `main` before a merge. Some minor comments follow.

File: peepholelib/models/svd.py [hunk 2]
```diff
@@ -8,8 +11,7 @@
 from tensordict import TensorDict
 from tensordict import MemoryMappedTensor as MMT
 
-def c2s(input_shape, weight, bias, stride=(1, 1), padding=(0, 0), dilation=(1,1), device='cpu', verbose=False, warns=True):
-
+def c2s(input_shape, weight, groups, bias, stride=(1, 1), padding=(0, 0), dilation=(1,1), device='cpu', verbose=False, warns=True):
```
Comment (lsrosa) [inline peepholelib/models/svd.py:14]: at this point we should pass the layer and get `weight, bias, stride, padding, dilation, groups` from the layer object.

**Review by lsrosa — CHANGES_REQUESTED** [review]

**Review by lsrosa — APPROVED** [review]
